ZAPD/ZLimb.cpp

```diff
@@ -240,11 +249,24 @@ std::string ZLimb::GetBodySourceCode() const
 	}
 	else
 	{
+		std::string childStr;
+		std::string siblingStr;
```

Comment:
Any possibility of using C strings?

Reply:
I mean I could, but why do it here when it we use C++ `std::strings` everywhere else?

Reply:
I switched to C strings in a few places where the extra functionality wasn't needed (Like in ZDisplayList). It saves the cost of allocation and constructing the whole string object.

Reply:
However if using a C string will be just as costly (If it say needs to do a lot more manual string manipulation) I'd say leave it.

Reply:
I think it would just make the code harder to read, since it would need to interface with the C++ strings that we are already using. 
Also, I don't think the effect of changing specifically this to C strings would be almost insignificant. A proper measurement should be done first